package.json

```diff
@@ -51,7 +50,7 @@
     "p-queue": "3.1.0",
     "slpjs": "^0.16.2",
     "traverse": "^0.6.6",
-    "zeromq": "^4.6.0"
+    "zeromq": "^5.1.0"
```

Comment:
The only breaking change is that Node.js 4 is no longer supported but SLPDB already requires Node.js > 8.15